packages/twenty-server/src/engine/metadata-modules/remote-server/utils/validate-remote-server-input.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { BadRequestException } from '@nestjs/common';
+
+const INPUT_REGEX = /^([A-Za-z0-9\-_.@]+)$/;
+
+export const validateObjectAgainstInjections = (input: object) => {
+  for (const [key, value] of Object.entries(input)) {
+    // Password are encrypted so we don't need to validate them
+    if (key === 'password') {
+      continue;
+    }
+
+    if (!value || !INPUT_REGEX.test(value.toString())) {
```

Comment:
sorry I think there is a misunderstood there. I don't think this is the role of this function to check if the value is not empty. If no value, just continue. If value, call `validateStringAgainstInjections(value.toString())`